### PR TITLE
EG-2016: bringing pages into line with Enterprise 4.5 node-administration.md

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md [hunk 1]
```diff
@@ -207,14 +207,13 @@ be configured to collect data from Jolokia and write to DataDog web api.
 In order to ensure that a Jolokia agent is instrumented with the JVM run-time, you can choose one of these options:
 
 
-* Specify the Node configuration parameter `jmxMonitoringHttpPort` which will attempt to load the jolokia driver from the `drivers` folder.
-The format of the driver name needs to be `jolokia-jvm-{VERSION}-agent.jar` where VERSION is the version required by Corda, currently 1.6.1.
-* Start the node with `java -Dcapsule.jvm.args="-javaagent:drivers/jolokia-jvm-1.6.1-agent.jar=port=7777,host=localhost" -jar corda.jar`.
+* Specify the Node configuration parameter `jmxMonitoringHttpPort` which will attempt to load the jolokia driver.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md:210]:
```suggestion
* Specify the Node configuration parameter `jmxMonitoringHttpPort` which will attempt to load the Jolokia driver.
```

File: content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md [hunk 2]
```diff
@@ -106,14 +106,13 @@ be configured to collect data from Jolokia and write to DataDog web api.
 In order to ensure that a Jolokia agent is instrumented with the JVM run-time, you can choose one of these options:
 
 
-* Specify the Node configuration parameter *jmxMonitoringHttpPort*.
-* When using the launcher, add the line *-javaagent:../../drivers/jolokia-jvm-1.6.0-agent.jar=port=7777,host=localhost* to the *[JVMOptions] sections of the `launcher/app/launcher.cfg*. Make sure to place the Jolokia agent that you specify there into the *drivers* folder.
-* Start the node with *java -jar corda.jar -javaagent:drivers/jolokia-jvm-1.6.0-agent.jar=port=7777,host=localhost*.
+* Specify the Node configuration parameter `jmxMonitoringHttpPort` which will attempt to load the jolokia driver.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md:109]:
```suggestion
* Specify the Node configuration parameter `jmxMonitoringHttpPort` which will attempt to load the Jolokia driver.
```

File: content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md [hunk 3]
```diff
@@ -207,14 +207,13 @@ be configured to collect data from Jolokia and write to DataDog web api.
...
+* Start the node with `java -Dcapsule.jvm.args="-javaagent:path/to/jolokia-jvm-{VERSION}-agent.jar=port=7777,host=localhost" -jar corda.jar` where `path/to/jolokia-jvm-{VERSION}-agent.jar` is the path to the driver, and `{VERSION}` is the version required by Corda, currently 1.6.1.
 
 The following JMX statistics are exported:
 
 
-* Corda specific metrics: see node-metrics for a list
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md:217]: add full-stops

File: content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md [hunk 4]
```diff
@@ -106,14 +106,13 @@ be configured to collect data from Jolokia and write to DataDog web api.
...
+* Start the node with `java -Dcapsule.jvm.args="-javaagent:path/to/jolokia-jvm-{VERSION}-agent.jar=port=7777,host=localhost" -jar corda.jar` where `path/to/jolokia-jvm-{VERSION}-agent.jar` is the path to the driver, and `{VERSION}` is the version required by Corda, currently 1.6.1.
 
 The following JMX statistics are exported:
 
 
-* Corda specific metrics: flow information (total started, finished, in-flight; flow duration by flow type), attachments (count)
+* Corda specific metrics: see [Node metrics](../../node-metrics.md) for a list
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md:115]: add full-stops

File: content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md (hunk 3 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.4/node/operating/node-administration.md:217]: Done

File: content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md (hunk 4 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.5/node/operating/monitoring-logging.md]: Done

**Pull request merged**